images/Dockerfile.nvidia-orin-nx

```diff
@@ -0,0 +1,285 @@
+	nvidia-l4t-3d-core \
+	nvidia-l4t-bootloader \
+	nvidia-l4t-apt-source \
+	nvidia-l4t-camera \
+	nvidia-l4t-configs \
+	nvidia-l4t-core \
+	nvidia-l4t-cuda-utils \
+	nvidia-l4t-cuda \
+	nvidia-l4t-display-kernel \
+	nvidia-l4t-firmware \
+	nvidia-l4t-gbm \
+	nvidia-l4t-init \
+	nvidia-l4t-initrd \
+	nvidia-l4t-jetson-io \
+	nvidia-l4t-kernel-dtbs \
+	nvidia-l4t-kernel-headers \
+	nvidia-l4t-kernel-oot-headers \
+	nvidia-l4t-kernel-oot-modules \
+	nvidia-l4t-kernel \
+	nvidia-l4t-nvfancontrol \
+	nvidia-l4t-nvml \
+	nvidia-l4t-nvpmodel-gui-tools \
+	nvidia-l4t-nvpmodel \
+	nvidia-l4t-nvsci \
+	nvidia-l4t-oem-config \
+	nvidia-l4t-openwfd \
+	nvidia-l4t-optee \
+	nvidia-l4t-pva \
+	nvidia-l4t-tools \
+	nvidia-l4t-vulkan-sc-dev \
+	nvidia-l4t-vulkan-sc-sdk \
+	nvidia-l4t-vulkan-sc \
+	nvidia-l4t-xusb-firmware \
+	cuda-cudart-12-6 \
+	cuda-cudart-dev-12-6 \
+	cuda-nvcc-12-6 \
+	cuda-nvdisasm-12-6 \
+	cuda-nvml-dev-12-6 \
+	cuda-nvprune-12-6 \
+	cuda-nvrtc-12-6 \
+	cuda-nvrtc-dev-12-6 \
+	cuda-nvtx-12-6 \
+	libcublas-12-6 \
+	libcublas-dev-12-6 \
+	jetson-gpio-common \
+	python-jetson-gpio \
+	python3-jetson-gpio \
+	nvidia-container-toolkit-base \
+	nvidia-container-toolkit \
+        && rm -rf /var/lib/apt/lists/*
+
+# Move CUDA out of the way to /opt so kairos can occupy /usr/local without workarounds
+RUN update-alternatives --remove-all cuda && \
+    update-alternatives --remove-all cuda-12 && \
+    mv /usr/local/cuda-12.6 /opt/cuda-12.6 && \
+    update-alternatives --install /opt/cuda cuda /opt/cuda-12.6 1 && \
+    update-alternatives --install /opt/cuda-12 cuda-12 /opt/cuda-12.6 1
+
+# change mountpoint for l4t usb device mode, as rootfs is mounted ro
+# /srv/data is made through cloud-config
+RUN sed -i -e 's|mntpoint="/mnt|mntpoint="/srv/data|' /opt/nvidia/l4t-usb-device-mode/nv-l4t-usb-device-mode-start.sh
+
+# init kairos
+RUN --mount=type=bind,source=/kairos-init,target=/kairos-init,from=quay.io/kairos/kairos-init:v0.6.1 \
+    echo "" > /boot/extlinux.conf && \
+    echo 'omit_dracutmodules+=" iscsi "' > /etc/dracut.conf.d/iscsi.conf && \
+    # We've already installed this sources list manually somewhere up above, \
+    # and it conflicts with Kairos \
+    rm -rf /etc/apt/sources.list.d/nvidia-l4t-apt-source.list && \
+    /kairos-init -l info -s install \
+	-m "${KAIROS_MODEL}" -t "${KAIROS_TRUSTED_BOOT}" -k "${KAIROS_KUBERNETES_DISTRO}" \
+	--k8sversion "${KAIROS_KUBERNETES_VERSION}" \
+	--version "${KAIROS_VERSION}" \
+	--skip-step installKernel,installPackages && \
+    /kairos-init -l info -s init -m "${KAIROS_MODEL}" -t "${KAIROS_TRUSTED_BOOT}" -k "${KAIROS_KUBERNETES_DISTRO}" \
+	--k8sversion "${KAIROS_KUBERNETES_VERSION}" \
+	--version "${KAIROS_VERSION}" && \
+    /kairos-init validate
+
+ARG K3S_VERSION="v1.28.5+k3s1"
+ARG BASE_VERSION="36.4.4"
+ARG RELEASE
+
+# iscsid causes delays on the login shell, and we don't need it, so we'll
+# disable it.
+RUN systemctl disable iscsi open-iscsi iscsid.socket
+
+# install k3s
```

Comment:
I'm not sure about this section, since it goes in a different direction than the standard approach, wdty @kairos-io/maintainers